app/assets/stylesheets/application.scss

```diff
@@ -56,9 +63,55 @@ h6 {
   h3 {color: black;
     text-transform: uppercase;
     font-style: normal;
+    a {
+      color: black;
+    }
   }
   h6 {color: #999;
     border-bottom: 1px solid #999;
   }
 
   }
+
+
+.featured-three{
+  background-color: #b9b9b9;
```

Comment:
Color literals like `#b9b9b9` should only be used in variable declarations; they should be referred to via variable everywhere else.